### ISSUE TITLE
Unable to exclude multiple packages in yum.conf

### ISSUE THREAD

**Issue by nmaludy** [voxpupuli/puppet-yum#80]
## Affected Puppet, Ruby, OS and module versions/distributions
- Puppet: 3.8.7
- Ruby: ruby 2.0.0p648 (2015-12-16) [x86_64-linux
- Distribution: Red Hat Enterprise Linux Server release 7.4 (Maipo)
- Module version: 0.10.0

## How to reproduce (e.g Puppet code you use)

I'm trying to exclude multiple packages in `/etc/yum.conf`. The man page for `yum.conf` says that this must be a space separated list of values, so i tried the following:

``` puppet
yum::config { 'exclude':
  ensure => 'openmotif* compat-libstdc*'
}
```

## What are you seeing

Only the first element is being put into the config.

``` shell
[main]
exclude=openmotif*
cachedir=/var/cache/yum/$basearch/$releasever
keepcache=0
debuglevel=2
logfile=/var/log/yum.log
exactarch=1
obsoletes=1
gpgcheck=1
plugins=1
installonly_limit=3
```

## What behaviour did you expect instead

A config file with the full string i specified for exclude: 

``` shell
[main]
exclude=openmotif* compat-libstdc*'
cachedir=/var/cache/yum/$basearch/$releasever
keepcache=0
debuglevel=2
logfile=/var/log/yum.log
exactarch=1
obsoletes=1
gpgcheck=1
plugins=1
installonly_limit=3
```

**Comment by joekohlsdorf** [comment]
This was fixed by commit 80af8a4a8f3275f5239f9e746e2981ca1b7476d6, issue can be closed.

**Comment by bastelfreak** [comment]
Thanks for letting us know @joekohlsdorf !
